app/Http/Controllers/GalleryController.php

```diff
@@ -42,15 +42,23 @@ function ShowAllYearsOfGallerys(){
 
 
     /// Dit is voor een andere user story, deze wordt later gemaakt. Jira: S8S-32 en S8S-33///
+
+    public function index() 
+    {
+        $albums = Album::all();
+        return view('Gallery.index', compact('albums'));
+    }
+
     public function create()
     {
-        return view('Gallery.aanmakenAlbum');
+        return view('Gallery.create');
     }
 
     public function store(Request $request)
     {
         $request->validate([
-            'title' => 'required',
+            'title' => 'required|max:255',
+            'description' => 'required|max:999',
             'date' => 'required'
```

Comment:
misschien dat de datum niet in de toekomst kan zijn?

Reply:
fixed